src/scala/org/pantsbuild/zinc/compiler/Settings.scala

```diff
@@ -47,7 +48,8 @@ case class Settings(
   compileOrder: CompileOrder        = CompileOrder.Mixed,
   sbt: SbtJars                      = SbtJars(),
   _incOptions: IncOptions           = IncOptions(),
-  analysis: AnalysisOptions         = AnalysisOptions()
+  analysis: AnalysisOptions         = AnalysisOptions(),
+  creationTime: Long                = System.currentTimeMillis()
```

Comment:
This isn't a good default for this arg: would recommend `0` (the unix epoch) or something...

Reply:
I think if we want the jars to be deterministic (each JAR entry to have the same creation time), the default needs to be consistent for each JAR created but not the same as every other JAR created. lmk if I misunderstand

Reply:
> but not the same as every other JAR created

Ideally "when" you create the jar does not matter at all. For the same inputs, creating the same jar twice (one+ second apart) should result in the exact same jar. Hence 0 here.